hadoop-common-project/hadoop-common/src/site/markdown/FileSystemShell.md

```diff
@@ -542,6 +542,8 @@ Options:
 * `-l` : Allow DataNode to lazily persist the file to disk, Forces a replication
  factor of 1. This flag will result in reduced durability. Use with care.
 * `-d` : Skip creation of temporary file with the suffix `._COPYING_`.
+* `-q <threadPool queue size>` : Number of threadPool queue size to be used, 
```

Comment:
here, too.

Reply:
Thanks to @jojochuang  comment, I will make some changes later.